### DISCUSSION TITLE
关于捐助被退款的事儿

### DISCUSSION THREAD

**Discussion by KngStr** [Remix-Design/RemixIcon#472] [General]
大佬好，
今年发现了remix，感觉很好。
昨天想支持一下，就支付宝捐助了500。
上午发现被退款了，特来问问怎么回事儿。
谢谢

**Comment by xiaochunjimmy** [comment]
金额有点大，毕竟很久没更新了，内心难安；精神支持已经到位了，万分感谢！会更新的！！

**Reply by KngStr** [reply]
大佬客气了。有在产品中用到，图标看着很舒心。这金额真不多，本来想捐一千呢，奈何实力有限。
这次就先捐这些吧，以后只要在用，我就会时不时来捐助的。
更新不更新，也已经有很多出色的图标了。
大佬不必推辞，这点钱说多不多，聊表谢意
再次感谢

**Reply by xiaochunjimmy** [reply]
好！那我不客气了！有其他需要帮助的地方直接找我就好

**Reply by KngStr** [reply]
哈哈，好的

**Comment by xiaochunjimmy** [comment]
今年（2023）愚人节这边我们发布了图标的更新，并且伴随着新的网站版本。
到目前图标也陆续更新到了 2704 个，希望可以对得起粉丝。

讲真，做开源其实很大程度出于情怀，用户多了之后又处于热爱和责任，但实际打赏的人寥寥可数，即便 8.8 元的打赏，在很多用户心中都是没必要的，所以我们也非常感谢那些以巨额姿态支持过我们的用户！再次表示感谢！

**Comment by KngStr** [comment]
理解理解。我自己也是程序员出身，开源界也是多少有点涉猎的。
大的氛围还是没起来，你们这么支持开源的世界，我还是很佩服的。
虽然我用的图标不多，但你们做的图标我个人是真的很喜欢。
巨额是算不上啦。聊表心意，表达我心中的喜欢和我的支持。

冒昧的问一句，你们是否接单兼职的设计活？我们偶尔会有点设计logo之类的需求。

**Reply by xiaochunjimmy** [reply]
感谢！

关于是否接单设计私活，还是取决于需求具体情况来评估。如果需求比较实用，并且不算太复杂的话，应该没问题的；比较担心的是甲方需求比较复杂，想要在一个图形上传递很多信息并且来回修改，或者还需要一套完整的 VI 手册，这种 case 暂时不倾向于接，时间成本确实有点大，我们目前没办法分出太多精力承接这样的需求

**Comment by KngStr** [comment]
好的，明白。有类似需求的时候我会再来的。
再次感谢
